warrior/Framework/Utils/cli_Utils.py

```diff
@@ -595,6 +595,10 @@ def send_commands_from_testdata(testdatafile, obj_session, **args):
                                                           result=result, response=response, system_name=system_name)
                 response_dict = _get_response_dict(details_dict, i, response,
                                                    response_dict)
+                if response_dict.values() is not None:
+                    sessid = system_name + "_td_response"
```

Comment:
here you have to append session name as well(if provided) to get the session id, use data_Utils.get_session_id method.

Reply:
Done @venkat1077